source/documentation/other-topics/custom-default-backend.md

```diff
@@ -0,0 +1,78 @@
+### Custom default-backend
+
+#### Background
+
+Cloud-Platform created [custom error pages][cloud-platform-custom-error-pages] to be used by the default backend for Nginx ingress-controller. The default backend is the default service that Nginx falls backs to if it cannot route a request successfully. Globally when the service in the Ingress rule does not have active endpoints, this default-backend service will handle the response by serving the cloud-platform custom default error page.
+
+However, some applications don’t want to use the cloud-platform custom default error pages from the Nginx ingress controller but need to be served their own custom error pages. This can be achieved by implementing own [custom default backend][customized-default-backend] in a namespace.
+
+#### Setup
+
+Custom default-backend can be managed inside a namespace by creating a custom default-backend service and deployment of the custom error pages container in your namespace. Configure the name of the service resource created that Nginx should use as the default backend for requests that don’t match any configured ingress rules (hostname and path combinations) using the annotation in your Ingress.
+
+##### Create custom error page
+First create a docker image containing custom HTTP error pages using the [example][ingress-nginx-custom-error-pages] from the ingress-nginx, or [simplified version][cloud-platform-custom-error-pages] of that created by the cloud platform team.
+
+##### Customised default backend
+Using this [custom-default-backend][customized-default-backend] example from ingress-nginx create a service and deployment of the error pages container in your namespace.
+
+```
+$ kubectl -n ${namespace} create -f custom-default-backend.yaml
```

Comment:
I think we should highlight that users should not be running these command manually. Their custom error pages configs should live in the folder from which they normally deploy to their namespace - we're just showing them how to do it manually, so they can play around while they get it right.

Reply:
Added a step to make it clear that it is to add it manually